### ISSUE TITLE
Displaying billboarded text

### ISSUE THREAD

**Issue by GearsAD** [rdeits/MeshCat.jl#29]
Any chance for supporting billboarded text (2D text that always faces the camera)?

Great library by the way! Similar to Drake without all the deps, nice.

**Comment by rdeits** [comment]
Yeah, I agree that would be nice. It's not currently supported in the underlying meshcat javascript library, though, so there'd have to be some work done. I can see a couple of options: 

1) Add an `set_text()` (or perhaps just `set_html()`) command to the meshcat javascript library and then bind that method from julia
2) Just compose the text you want with the meshcat window directly using WebIO.jl. 

The second option likely requires fewer changes (once #28 comes in), but might be less flexible. 

Can you tell me more about what you'd want to do with billboarded text?

**Comment by rdeits** [comment]
Ah, ok. Yes, if you want the text to live in the 3D world (but be rendered facing the camera), then that's definitely option (1). I'm not sure how to do that in threejs either, but I think the link you posted is the right place to start.

**Comment by ThatcherC** [comment]
Any chance adding some sort of text display will be easier now that #28 is merged? I'd love to be able to add some text tags to parts of my scenes but haven't found a good way to do it yet.
